src/DefaultBuilder/samples/SampleApp/Program.cs

```diff
@@ -15,13 +14,14 @@ namespace SampleApp
 {
     public class Program
     {
-        public static async Task Main(string[] args)
+        public static void Main(string[] args)
         {
-            await using var webApp = WebApplication.Create(args);
+            var app = WebApplication.Create(args);
 
-            webApp.MapGet("/", (Func<string>)(() => "Hello, World!"));
+            app.UseStaticFiles();
```

Comment:
I guess you manually tested this?

Reply:
Yes. I can write a manual test. There aren't any e2e tests in DefaultBuilder that don't hit the network stack so I was a little lazy and just went with the unit tests and the sample.